src/test/scala/chiseltest/tests/FaultLocatorTest.scala

```diff
@@ -39,13 +39,13 @@ class FaultLocatorTest extends AnyFlatSpec with ChiselScalatestTester with Match
         c.out.setSinkClock(c.clock)
 
         c.in.valid.poke(true.B)
-        c.in.bits.poke(false.B)  // Have this be a data failure only
+        c.in.bits.poke(false.B) // Have this be a data failure only
         c.out.expectDequeueNow(true.B)
       }
     }
     // Only check the filename to avoid this being too brittle as implementation changes
-    exc.failedCodeFileNameAndLineNumberString.get should startWith ("DecoupledDriver.scala:")
-    exc.getMessage should include regex ("""\(lines in FaultLocatorTest\.scala:[^\)]*43.*\)""")
+    exc.failedCodeFileNameAndLineNumberString.get should startWith("DecoupledDriver.scala:")
+    (exc.getMessage should include).regex("""\(lines in FaultLocatorTest\.scala:[^\)]*43.*\)""")
```

Comment:
Yeah, the formatter did not do the right thing here either.

Reply:
Should I add more keyword in the PR: https://github.com/scalameta/scalafmt/pull/2252?